### PR TITLE
Interactive lists

### PR DESCRIPTION
**Pull request by see-why**
- Add a new JavaScript file (list.js) and import it as a module
- Add event listener to the checkbox (change)
- Update items object's value for completed key upon user actions.
- Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/list.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export function toggleCompleteTask(className) { /* eslint-disable-line no-unused-vars */
```
Comment (rloterh) [inline src/list.js]: - [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

File: src/list.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  if (className.style.textDecoration === 'line-through') {
+    className.style.textDecoration = 'none';
+  } else {
+    className.style.textDecoration = 'line-through';
+  }
+}
+
+export function strikeText(checked, className) {
```
Comment (rloterh) [inline src/list.js]: - [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Cy, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Modul import correctly implemented
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 3]
```diff
@@ -34,12 +42,21 @@ class TodoList {
       checkInput.checked = item.completed;
       checkInput.className = 'todo-list-ul-li-box';
 
+      checkInput.addEventListener('change', (e) => {
+        const textInput = e.target.parentNode.lastChild;
+        toggleCompleteTask(textInput); /* eslint-disable-line no-undef */
```
Comment (Lameck1) [inline src/index.js]: - [OPTIONAL] You can also undo this linter disabling, I found out it has no effect and your code will still run fine.

**Review by Lameck1 — APPROVED** [review]
Hi, @see-why,

Thanks for working on the previously requested changes.
This milestone is now complete and good luck with the next one.
You are free to merge this branch :shipit:

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 3 above)
Comment (see-why) [reply src/index.js]: it has been removed...thanks a lot.

**Pull request merged**